src/bp/ui-shared/package.json

```diff
@@ -9,7 +9,8 @@
   },
   "scripts": {
     "start": "node ./webpack.config.js --watch",
-    "build": "cross-env NODE_ENV=production node ./webpack.config.js --compile --nomap"
+    "build": "cross-env NODE_ENV=production node ./webpack.config.js --compile --nomap & node-sass src/theme/main.scss dist/theme.css --importer=node_modules/node-sass-tilde-importer",
+    "scss": "node-sass src/theme/main.scss dist/theme.css --importer=node_modules/node-sass-tilde-importer"
```

Comment:
Can the build command use directly yarn scss instead, so we don't have to duplicate commands ?